client/src/App.jsx

```diff
@@ -56,7 +54,7 @@ function App() {
             clientId={clientID}
             domain="https://whisper.kinde.com"
             logoutUri={window.location.origin}
-            redirectUri={window.location.origin + '/login'}
+            redirectUri={window.location.origin}
```

Comment:
this has nothing to do with your change, revert back everything in this file

Reply:
Sorry about that resolved, when i was having issues with login in using kinde that was when i removed the plus login but forgot to undo